flask_common/enum.py

```diff
@@ -13,19 +13,34 @@ class MyEnum(Enum):
     is prone to typos.
     """
 
+    __values = []
```

Comment:
Add comment that these are containers for caching the results of `values()`/`choices()`.